website/src/app/[lang]/[region]/(blue-theme)/donate/success/stripe/[session]/page.tsx

```diff
@@ -34,7 +34,7 @@ export default async function Page({ params }: StripeSuccessPageProps) {
 	}
 
 	if (contributorResult.data && !contributorResult.data.needsOnboarding) {
-		redirect(`/${lang}/${region}/dashboard/contributions`);
+		redirect(`/${lang}/${region}/dashboard/subscriptions`);
```

Comment:
i think the idea is to check if a contributor did a one time donation and if yes redirect to contributions and if not redirect to subscriptions. 
I don't know if its even possible?